_docs-sources/pipelines/maintain/extending.md

```diff
@@ -50,27 +61,36 @@ infrastructure-deployer --aws-region "us-east-1" -- docker-image-builder build-d
 
-Depending on your use case, you may need to modify the allow-list to only allow a pre-defined list of branch names.
+Depending on your use case, you may need to modify the `allowed_apply_git_refs` attribute to update the allow-list of branch names that can kick off the `plan` and `apply` jobs.
+
+For example, a common configuration for `apply` jobs is to specify that this job can only run on the `main` branch:
+```tf
+allowed_apply_git_refs = ["main", "origin/main"]
+```
 
-### RefArch
+<Tabs groupId="deployment-type">
+<TabItem value="RefArch" label="RefArch" default>
 
-Open `shared/<your region>/mgmt/ecs-deploy-runner/terragrunt.hcl` and update the values the `allowed_apply_git_refs` attribute for the job configuration you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
+If you've deployed Pipelines as a part of your Reference Architecture, open `shared/<your region>/mgmt/ecs-deploy-runner/terragrunt.hcl` and update the values in the `allowed_apply_git_refs` attribute for the job configuration you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
 
 Run `terragrunt plan` to inspect the changes that will be made to your pipeline. Once the changes have been reviewed, run `terragrunt apply` to deploy the changes.
 
-### Standalone
+</TabItem>
+<TabItem value="Standalone" label="Standalone">
 
 If you've deployed Pipelines as a standalone framework using the `ecs-deploy-runner` service in the Service Catalog, you will need to locate the file in which you've defined a module block sourcing the `ecs-deploy-runner` service.
 
 By default, the `ecs-deploy-runner` service from the Service Catalog allows any git ref to be applied. After you locate the module block for `ecs-deploy-runner`, modify the `allowed_apply_git_refs` attribute for the job configuration that you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
 
 Run `terraform plan` to inspect the changes that will be made to your pipeline. Once the changes have been reviewed, run `terraform apply` to deploy the changes.
+</TabItem>
+</Tabs>
 
-## Adding script arguments
+## Adding scripts that can be run in Pipelines
 
 The `deploy-runner` Docker image for Pipelines only allows scripts within a single directory to be executed in the ECS task as an additional security measure.
 
-By default, the `deploy-runner` ships with three scripts - one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
+By default, the `deploy-runner` ships with three scripts — one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
```

Comment:
Nit: use unordered list.

Reply:
@oredavids I discussed the unordered list vs in paragraph approach live. In these cases Eben was okay with the in paragraph approach.